src/components/Form/Accordion/Accordion.test.jsx

```diff
@@ -333,4 +346,40 @@ describe('The accordion component', () => {
     expect(component.find('.append-button button').length).toEqual(1)
     expect(component.find('.addendum').length).toEqual(0)
   })
+
+  it('validates a validator function', () => {
+    const ValidValidator = class {
+      isValid () {
+        return true
+      }
+    }
+    const InvalidValidator = class {}
+    let items = [
+      {
+        func: ValidValidator,
+        expected: true
+      },
+      {
+        func: InvalidValidator,
+        expected: false
+      },
+      {
+        func: {},
+        expected: false
+      },
+      {
+        func: () => {},
+        expected: false
+      }
+    ]
```

Comment:
See the comment here about additional tests:  https://github.com/18F/e-QIP-prototype/pull/92#discussion_r137691465